.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -42,19 +51,24 @@ E.g. `explain select 1`
 +---------------+--------------------------------------+
 ```
 
-**spice, spiced, OS info**
+### spice, spiced, OS info
+
 - spice version: `spice version`
 - spiced version: `spiced --version`
 - OS info: `uname -a`
 
 **Have you tried this on the latest `trunk` branch?**
 
-**If possible, run `spiced` with DEBUG log level**
+- [ ] Yes
+- [ ] No
 
+**If possible, run `spiced` with DEBUG log level**
 By setting the environment variable `SPICED_LOG="spiced=DEBUG,runtime=DEBUG,data_components=DEBUG,cache=DEBUG"`
```

Comment:
isn't this out of date now?, Currently its https://github.com/spiceai/spiceai/blob/eb3b368eea0e64efc61af8f05bd8511558b2e064/bin/spiced/src/tracing.rs#L38